src/molecules/account-menu.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from './account-menu.module.css';
+
+import { forwardRef } from "react";
+
+const AccountMenu = ({ isLoggedIn, userName, userIcon, onLogout, onLogin }, ref) => {
```

Comment:
```suggestion
import { Ref, forwardRef } from "react";

export interface AccountMenuProps {
  user: {
    name: string;
    initials: string;
  } | null;
  onLogout: () => void;
  onLogin: () => void;
}

const AccountMenu = (
  { user, onLogout, onLogin }: AccountMenuProps,
  ref: Ref<HTMLDivElement>,
) => {
```
コンポーネントの引数も修正しましょう. ログインされているときは `user` に `name` と `initials` が入っていて, ログインされていないときは `user` が `null` になっているようなパラメータ定義はいかがでしょうか.

Reply:
8a4986f で修正が巻き戻っているようです